db/migrate/20220202122417_add_parent_id_to_miq_requests.rb

```diff
@@ -0,0 +1,9 @@
+class AddParentIdToMiqRequests < ActiveRecord::Migration[6.0]
+  class MiqRequest < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
```

Comment:
Unless you're using the model name in a data migration (like if you were doing `MiqRequest.in_my_region.update_all(:parent_id => nil)` or something like that) you don't need to define this